weblate/templates/snippets/project/billing-libre-trial.html

```diff
@@ -8,10 +8,10 @@
   {% if user_can_view_billing %}
     {# project admin can request approval #}
     <a href="{{ object.billing.get_absolute_url }}" class="btn btn-primary pull-right flip">{% trans "Request approval for Libre hosting" %}</a>
-    {% trans "This project is in the trial period, and is publicly accessible. You can lock it and post an announcement in case you want to avoid contributions until you finish the setup." %}
+    {% trans "This trial project is publicly accessible. Post an announcement and/or lock it to avoid contributions until you finish the setup." %}
   {% else %}
     {# users are warned that the setup is not yet completed #}
-    {% trans "This project is in the trial period, so be cautious before you contribute. Things like license or repository can still change before being approved as a libre project." %}
+    {% trans "Back up your translations till the project is fully set up and accepted as libre." %}
```

Comment:
```suggestion
    {% trans "Back up your translations till the project is fully set up and approved as libre." %}
```

For consistency, I would suggest using the verb `approved` here.

See https://github.com/WeblateOrg/weblate/blob/9de4393a1974929992f964cdf01f3303ab883dd8/weblate/templates/billing/detail.html#L124